core/sql/executor/ExExeUtilGetStats.cpp

```diff
@@ -2319,47 +2321,38 @@ short ExExeUtilGetRTSStatisticsTcb::work()
       {
         if (measStatsItems_ == NULL)
         {
-          maxMeasStatsItems_ = 35;
+          maxMeasStatsItems_ = 26;
```

Comment:
It looks like I looked at the wrong file ResStatisticsStatement.cpp when I responded to Arvind. Hence I didn't catch the issue with duplicate index 22

measStatsItems_[22].statsItem_id = SQLSTATS_SCRATCH_BUFFER_BLOCKS_READ;
measStatsItems_[22].statsItem_id = SQLSTATS_SCRATCH_BUFFER_BLOCKS_WRITTEN; measStatsItems_[23].statsItem_id = SQLSTATS_SCRATCH_READ_COUNT;
measStatsItems_[24].statsItem_id = SQLSTATS_SCRATCH_WRITE_COUNT;

I will fix it